DolbyIO/Source/DolbyIoConferenceModule/Private/DolbyIoConference.cpp

```diff
@@ -27,8 +27,11 @@ void ADolbyIoConference::BeginPlay()
 	{
 		FirstPlayerController = World->GetFirstPlayerController();
 	}
-
 	CppSdk->SetObserver(this);
+	if (!Token.IsEmpty())
+	{
+		CppSdk->RefreshToken(Token);
```

Comment:
Now that this is here, it would be useful to restore the documentation in the header that was previously there, mentioning that the object initializes the SDK at BeginPlay if a valid token is present.